tools/api-builder/angular.io-package/processors/addJadeDataDocsProcessor.js

```diff
@@ -130,10 +135,10 @@ module.exports = function addJadeDataDocsProcessor() {
               title: exportDoc.name,
               docType: exportDoc.docType,
               exportDoc: exportDoc,
-              stability: stability,
-              howToUse: howToUse,
-              whatItDoes: whatItDoes,
-              security: security
+              stability: escapeQuotes(stability),
+              howToUse: escapeQuotes(howToUse),
+              whatItDoes: escapeQuotes(whatItDoes),
+              security: escapeQuotes(security)
```

Comment:
yeah. I agree but I didn't know how to do it in a template :) I left a comment on your pr